SkyBlock/README.md

```diff
@@ -0,0 +1,358 @@
+
+### List of island Types
+
+A list of island types shall be added in the near future.
+
+## Money
+
+As you progress, you will need to use money and make transactions. Money is stored in one of two places: wallet or bank. You can see the balance of both on the right hand side of your screen.
+
+### Interest
+
+For money stored in your bank, you recieve an interest. The interest is credited every hour you are online. The rate of interest is not revealed and keeps changing.
+
+## Challenges
+
+Challenges let you complete various tasks and recieve rewards. These rewards are in the form of items and experience. These items are useful in your journey to beat the gamemode. You can check your challenges using the command [`/is challenges`](#challenges).
+
+### List of Challenges
+
+This list shall be added in the near future.
+
+## Settings
+
+Settings allow you to customize your island. You can view and change your settings using the command [`/is settings`](#settings).
+
+There are currently 59 Protection Settings, 6 General Settings and 17 SkyBlock Island Settings that can be applied to the SkyBlock Island. Each of which ENABLE or DISABLE access to one or more types of players: Visitor, Coop, Trusted, Member, Sub-Owner and Owners for that island.
+
+### List of settings
+
+<!-- The below table was copy pasted from the BentoBox wiki -->
+
+<table width="100%" align="center"> <tbody><tr> <td align="left" valign="top"><b>Settings</b></td> <td align="left"><b>Flags to enable or disable</b></td> </tr> <tr> <td align="left" valign="top">Protection Settings</td> <td align="left">Animal riding, Anvils, Armor stands, Beacons, Beds, Break blocks, Breed animals, Brewing stands, Buckets, Buttons, Chorus fruits, Collect lava, Collect water, Egg throwing, Enchanting Table, End Portal, EnderPearls, Experience pickup, Fire, Fire extinguish, Fish scooping, Frost walker, Furnace, Gates, Hurt animals, Hurt monsters, Hurt villagers, Item drop, Item pickup, Jukebox use, Leash use, Lever use, Lock island, Milking, Mount inventory, Name tags, Nether Portal, Note block, Place blocks, Pressure Plates, Redstone items, Shearing, Spawn eggs, Next Page (sign), TNT damage, Trample crops, Trap doors, Turtle Eggs, Use doors, Villager trading, Workbenches, Use containers, Use dispensers, Use droppers, Use hoppers, Use Item Frames, Throw Potions, Previous Page (sign)</td> </tr> <tr> <td align="left" valign="top">General Settings</td> <td align="left">Animal spawning, End PVP, Fire spread, Monster spawning, Nether PVP, Overworld PVP</td> </tr> <tr> <td align="left" valign="top">BSkyBlock Settings</td> <td align="left">Chest damage, Clean Super Flat, Coarse dirt tilling, Creeper damage, Creeper griefing, Ender Chests, Enderman griefing, Enter/Exit messages, Island respawn, Offline Redstone, Piston Push, Remove monsters, Obsidian Scooping, Feather Fall Teleporting, Command Ranks, Invincible Visitors, Limit mobs to island</td> </tr> </tbody></table>
```

Comment:
Don't use HTML in markdown. please.

Reply:
HTML tables are easier and more intuitive than md.